genie-web/src/main/java/com/netflix/genie/web/tasks/job/JobCompletionService.java

```diff
@@ -200,8 +193,11 @@ private Job getJob(final String jobId) throws GenieException {
         return this.jobSearchService.getJob(jobId);
     }
 
-    private Void updateJob(final Job job, final JobFinishedEvent event)
-        throws GenieException {
+    private Void updateJob(
+        final Job job,
+        final JobFinishedEvent event,
+        final Map<String, String> tags
```

Comment:
Isnt it better to return the status of the job rather than sending in the map?

Reply:
That might be but I suspect he (@mprimi) put it in this way in case he needed to provide more tags within the method.